packages/bindgen/src/templates/node-wrapper.ts

```diff
@@ -59,8 +62,16 @@ export function generate({ spec: rawSpec, file }: TemplateContext): void {
     // We know that node always has real WeakRefs so just use them.
     export const WeakRef = global.WeakRef;
   `);
+  browserLines.push(`
+    /*global window, FinalizationRegistry*/
+    import Module from "./realm-js-wasm.js";
+    const nativeModule = await Module(); // loading WASM 
+    nativeModule.browserInit();
```

Comment:
I think this is a good spot to inject things like `UUID` and `Decimal128` on `Module` and recall them with `val::module_property()` in C++ instead of relying on the injectables mechanic.

Reply:
I'd rather we stay consistent between the engines for now, unless there is a good reason to diverge. Is there some benefit to injecting them this way rather than using `injectInjectables`?

Reply:
`browserInit();` exists solely to compensate for the lack of "engine registration phase" (i.e `NODE_API_NAMED_ADDON(realm_cpp, RealmAddon)` or  `realm_jsi_init` _which is called from_ `Java_io_realm_react_RealmReactModule_install`) which will trigger the creation of the `RealmAddon` instance.

Registering the JS type should still be done via `injectInjectables` IMO to keep the separation of concerns ...